### PR TITLE
Add rubocop_base.yml for common rubocop rules

### PR DESCRIPTION
**Pull request by lindgrenj6**
https://issues.redhat.com/browse/RHCLOUD-10237

Rather than making a new repo lets use api-common as the main repo. We will need to update ALL THE THINGS after this is merged though!

### CODE REVIEW

**Comment by eclarizio** [comment]
@lindgrenj6 I'm fine with this if we want to keep it in here instead of the new "manageiq-style" repo, but I think with this change we have to update rubocop on dippy-bot to a later version. I made a similar PR for Catalog: https://github.com/RedHatInsights/catalog-api/pull/844 but dippy-bot is also complaining about missing linter config files.

File: .rubocop_base.yml [hunk 1]
```diff
@@ -0,0 +1,292 @@
+require:
+  - rubocop-rails
+  - rubocop-performance
+
+AllCops:
+  NewCops: disable
+  Exclude:
+    - spec/manageiq/**/*
```
Comment (slemrmartin) [inline .rubocop_base.yml:8]: this is not needed

File: .rubocop_base.yml [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    - vendor/**/*
+  TargetRubyVersion: 2.5
```
Comment (slemrmartin) [inline .rubocop_base.yml:10]: can be switched to 2.6?

**Review by slemrmartin — COMMENTED** [review]
Just a few comments, otherwise LGTM (I'm not reviewing the rules, it's not a subject of this PR)

File: .rubocop_base.yml (hunk 2 above)
Comment (lindgrenj6) [reply .rubocop_base.yml]: Sure, updated.

**Comment by lindgrenj6** [comment]
@eclarizio yeah I think that is one thing we are going to try to do now/in the future. It would be really nice to upgrade to rubocop 1.0 since that was just released recently.

**Review by syncrou — APPROVED** [review]
I'm good with this. For anyone who wants to use the old MIQ rules ( at the new repo ) this change allows that. For those wishing to manage them internal to RedHatInsights, this change allows for that. LGTM 👍

**Comment by syncrou** [comment]
@eclarizio @lindgrenj6 with the questions about Dippy Bot, that doesn't hold this PR up does it?

**Pull request merged**